### PR TITLE
update model

### PR DESCRIPTION
**Pull request by osadolortech**

### CODE REVIEW

File: tweetapi/models.py [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
 
 # Create your models here.
+
+class ProfileModel(models.Model):
+    first_name = models.CharField(max_length=120,blank=True)
+    last_name = models.CharField(max_length=120,blank=True)
+    username = models.ForeignKey(User,on_delete=models.CASCADE,related_name='user')
+    Bio = models.CharField(max_length=150)
+    Location = models.CharField(max_length=60)
+    Birth_date = models.DateField(editable=True)
```
Comment (iamr0b0tx) [inline tweetapi/models.py]: ???????????????????????????????

**Review by iamr0b0tx — CHANGES_REQUESTED** [review]

File: tweetapi/models.py (hunk 1 above)
Comment (osadolortech) [reply tweetapi/models.py]: the design schema had this fields for the model just like how twitter profile is.

Comment (iamr0b0tx) [reply tweetapi/models.py]: Keep looking at it until you figure out what is wrong with it. I won't tell you

Comment (osadolortech) [reply tweetapi/models.py]: i should have had the blank = True to both location and bio field

Comment (osadolortech) [reply tweetapi/models.py]: should have used a PlainLocationField instead? that would require a google API key for full functionality.....a little hint would help sir

Comment (osadolortech) [reply tweetapi/models.py]: i think the bio,location and date_of_birth should not be among with the profile it should just have first_name,last_name, username.....

Comment (osadolortech) [reply tweetapi/models.py]: Sir a hint to help me out of this position I'm in total confusion on what to do I have been staring at this code for hours....

Comment (iamr0b0tx) [reply tweetapi/models.py]: It saddens me that even after all this time and learning I still have to correct variable name mistakes. Correct the variable names by using snake case. I won't correct you if this happens again.

Comment (osadolortech) [reply tweetapi/models.py]: ok sir, that totally skipped my mind. very sorry will fix right away

Comment (osadolortech) [reply tweetapi/models.py]: changes made sir

**Review by iamr0b0tx — APPROVED** [review]

**Pull request merged**